web/api/README.md

```diff
@@ -23,12 +18,16 @@ introduced.
 version: `change-api-version.sh 6.40.0`.
```

Comment:
Maybe we can change this to `./change-api-version.sh 6.40` for better clarity?

Reply:
No, we are using this form for versioning: <major>.<minor>.<patch>. I think we never incremented the patch version number so far, but who knows, maybe one time :smile: However I do not want to change this behaviour in this patch.